content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst

```diff
@@ -435,205 +476,250 @@ For these transactions it’s important to have into consideration the next requ
-next to the AFIP Authorization code. 
 
-.. image:: argentina/argentina_edi_17.png
+This tool incorporates in the vendor bill a new :guilabel:`Verify on AFIP` button located next to
+the :guilabel:`AFIP Authorization code`. 
+
+.. image:: argentina/verify-on-afip.png
    :align: center
+   :alt: Verify on AFIP.
 
-In case it’s not a valid AFIP authorization the value “Rejected” will be 
-displayed and the details of the validation will be added to the chatter.
+In case it's not a valid AFIP authorization the value :guilabel:`Rejected` will be displayed and
+the details of the validation will be added to the chatter.
 
-.. image:: argentina/argentina_edi_18.png
+.. image:: argentina/afip-auth-rejected.png
    :align: center
-
+   :alt: AFIP authorization Rejected.
 
 Special Use cases
 ~~~~~~~~~~~~~~~~~
+
 Untaxed Concepts
 ****************
-There are some transactions that include items that are not part of the VAT base amount, 
-this is commonly used in fuel and gasoline invoices. 
 
-The vendor bill will be registered using 1 item for each product that is part of the VAT
-base amount and an additional item to register the amount of  the Exempt concept:
+There are some transactions that include items that are not part of the VAT base amount, this is
+commonly used in fuel and gasoline invoices. 
+
+The vendor bill will be registered using 1 item for each product that is part of the VAT base
+amount and an additional item to register the amount of the Exempt concept:
 
-.. image:: argentina/argentina_edi_19.png
+.. image:: argentina/vat-exempt.png
    :align: center
+   :alt: VAT exempt.
 
 Perception Taxes
 ****************
-The vendor bill will be registered using 1 item for each product that is part of the 
-VAT base amount, the perception tax can be added in any of the product lines, as result 
-we will have one tax group for the VAT and one for the perception, the perception default 
-value is always 1.00. 
 
-.. image:: argentina/argentina_edi_20.png
+The vendor bill will be registered using 1 item for each product that is part of the VAT base
+amount, the perception tax can be added in any of the product lines, as result we will have one
+tax group for the VAT and one for the perception, the perception default value is always
+:guilabel:`0.10`.
+
+.. image:: argentina/vat-perception.png
    :align: center
+   :alt: VAT perception.
 
-You should use the pencil that is the next to the Perception amount to edit it 
-and set the correct amount. 
+To edit it and set the correct amount you should use the :guilabel:`Pencil` that is the next to the
+Perception amount.
 
-.. image:: argentina/argentina_edi_21.png
+.. image:: argentina/enter-perception-amount.png
    :align: center
+   :alt: Enter the perception amount.
    
-After this is done the invoice can be validated. 
-   
+Then invoice can be validated.
    
 Reports
 =======
 
 As part of the localization the next Financial reports were added:
```

Comment:
Suggestion on how to rewrite these lean introductions by providing more expository context, as well as navigation/placement information so the user doesn't need to rely on the images to make sense of what's being discussed.

```suggestion
As part of the localization installation, Financial reporting for Argentina was added inside the
:guilabel:`Accounting` dashboard. Access these reports by navigating to :menuselection:`Accounting
--> Reporting --> Argentinian Reports`
```